src/duktape/log.cr

```diff
@@ -8,8 +8,8 @@ require "log"
 require "json"
 
 Log.setup_from_env(
-  level: ENV.fetch("CRYSTAL_LOG_LEVEL", "INFO"),
-  sources: ENV.fetch("CRYSTAL_LOG_SOURCES", "duktape.*"),
+  log_level_env: "CRYSTAL_LOG_LEVEL",
```

Comment:
I'm temped to migrate over to the `LOG_LEVEL` environment variable name.

What do you think @kostya?

It looks like that's [the default value in Crystal core now](https://github.com/crystal-lang/crystal/blob/e68767e4937a542b65bcec6f5d12b52dff8c701b/src/log/setup.cr#L41). I'd like to keep this library as close to the convention in the community as standard library as possible.